### PR TITLE
Email communication image bug

### PR DESCRIPTION
**Pull request by yucohen**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/43288

## Description
Inline images were not shown in email HTML layout. I changed the attachment handling in MSGraphMail, changed displayEmailHtml and added 'get attachments' support in preProcessEmail for MSGraphMail

## Screenshots
image
image

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Review by Bargenish — CHANGES_REQUESTED** [review]
Nice work

* Add an RN and bump version for the `MicrosoftGraphListener` pack.
* Add a comment in the PreprocessEmail script in the  `get_attachments_using_instance` that attachments are downloaded by default when emails are fetched using the graph integrations.
* Add tests to the PreProcessEmail changes

File: Packs/MicrosoftGraphListener/ReleaseNotes/1_0_21.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### O365 Outlook Mail Single User (Using Graph API)
+- Fixed an issue where inline images were not getting into attachments list.
```
Comment (reutshal) [inline Packs/MicrosoftGraphListener/ReleaseNotes/1_0_21.md:4]: Lets be more specific here - please mention the scope when this happens -command/use case

File: Packs/MicrosoftGraphMail/ReleaseNotes/1_1_4.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### O365 Outlook Mail (Using Graph API)
+- Fixed an issue where inline images were not getting into attachments list.
```
Comment (reutshal) [inline Packs/MicrosoftGraphMail/ReleaseNotes/1_1_4.md:4]: same here

**Review by reutshal — APPROVED** [review]
Nice job 🚀

**Review by Bargenish — APPROVED** [review]

**Pull request merged**